scripts/generate-sdk/generate-sdk.sh

```diff
@@ -16,6 +22,31 @@ SCRIPTS_FOLDER="${SDK_REPO_LOCAL_PATH}/scripts"
 GENERATOR_VERSION="v6.6.0"
 GENERATOR_VERSION_NUMBER="${GENERATOR_VERSION:1}"
 
+if [[ -z ${GIT_HOST} ]]; then
+    echo "Git host is empty, default will be used."
+    GIT_HOST=${GIT_HOST_DEFAULT}
+fi
```

Comment:
This one also needs to be used in the call to the generator

Reply:
Added it to the generator call